recipes/gstlal-burst/meta.yaml

```
@@ -0,0 +1,106 @@
+{% set name = "gstlal-burst" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz"
+  sha256: 87c8c2a3691558afc7adaa4f4669c0a90a9fdf485bb2ca9df2e9ced7f8be95ed
+  patches:
+    # https://git.ligo.org/lscsoft/gstlal/issues/46
+    - missing-links.patch
+    # https://git.ligo.org/lscsoft/gstlal/issues/47
+    - gstaudio.patch
+
+build:
+  number: 0
+  skip: true  # [win or py>=30]
+  ignore_run_exports:
+    # header is included, but never used, reported upstream
+    - lalinspiral
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make  # [unix]
+    - pkg-config >=0.18.0  # [unix]
+  host:
+    - glib
+    - gobject-introspection >=1.30.0
+    - gsl
+    - gst-plugins-base
+    - gstlal >=1.5.0
+    - gstreamer
+    - lal >=6.19.0
+    - lalburst >=1.5.0
+    - lalinspiral
+    - numpy 1.14
```

Do you need to put a version here? I think the pins are [already 1.14 or higher](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L527-L529):
```suggestion
    - numpy
```

---

See https://github.com/conda-forge/staged-recipes/pull/10833#discussion_r378228818